custom-elements/custom-elements-registry/define.html

```diff
@@ -208,42 +215,38 @@
   // 17. If attributeChangedCallback is not undefined, and IsCallable(attributeChangedCallback) is false,
   // then throw a TypeError exception.
   [
-    'observedAttributes', // See step 8 above.
     'connectedCallback',
     'disconnectedCallback',
     'attributeChangedCallback',
-  ].forEach(name =>  {
-    test(() =>  {
+  ].forEach(name => {
+    test(() => {
       class C {
-        get [name]() { throw_rethrown_error(); }
+        get[name]() { throw_rethrown_error(); }
```

Comment:
Nit: I'd keep this space. For the non-computed case, `get observedAttributes() { }`, it's necessary, so it's nice to be consistent.